samples/config_helper_vulkan.h

```diff
@@ -44,6 +44,7 @@ class ConfigHelperVulkan : public ConfigHelperImpl {
   amber::Result CreateConfig(
       uint32_t engine_major,
       uint32_t engine_minor,
+      int32_t selected_device,
```

Comment:
This going to at least need to be added to config_helper_dawn or the build will fail, yea?

Reply:
Added the parameter to Dawn's CreateConfig. I wasn't able to build with Dawn support so there might still be something else I missed.